flask_validator/validator_engine.py

```diff
@@ -1,22 +1,17 @@
 import sys
 import traceback
+
 from functools import wraps
+
 from flask import current_app, request, jsonify, session
+
+from .error_bag import ErrorBag
 from .validators import validators
 from .exceptions import ValidatorAttributeError, ValidatorKeyError
-from .error_bag import ErrorBag
-
-
-class ValidatorEngine():
 
-    def __init__(self, app=None, db=None):
-        self.app = app
-        if app is not None and db is not None:
-            self.init_app(app, db)
 
```

Comment:
Removing the __init__ method and the init_app is not necessary as it will be used later.